chatbot_ner/setup_logs.py

```diff
@@ -0,0 +1,116 @@
+import collections
+from typing import Dict, Any, NamedTuple, List, Union, Optional
```

Comment:
F401 'typing.Union' imported but unused
F401 'typing.Optional' imported but unused

Reply:
resolved in latest commit